docs/topics/test-system.md

```diff
@@ -161,9 +165,44 @@ So Adam will write these docs once that task is merged and resolved.
 ## Updating
 
 All data is stored in volumes, so the easiest way to update a test system is to turn it off, recreate it using the steps above, just with different intputs, and then run `up` again.
-Writing the docs for this step is also blocked on https://phabricator.wikimedia.org/T298632 so that LocalSetting.php can be taken along for this ride.
-So Adam will write these docs once that task is merged and resolved.
+The one thing that needs copying over and mounting in the docker-compose file is the LocalSetting.php file for MediaWiki which on initial setup is created by the wikibase container and stored in the container only.
+
+That would look something like this...
+
+```sh
+TEST_SYSTEM=fedprops-previous
+
+cd /opt/test-systems/$TEST_SYSTEM
+sudo docker cp ${TEST_SYSTEM}_wikibase_1:/var/www/html/LocalSettings.php /tmp/LocalSettings-${TEST_SYSTEM}-$(date --iso).php
+sudo docker-compose -f docker-compose.yml -f docker-compose.extra.yml down
+cd /opt/test-systems
+mv ./$TEST_SYSTEM ./$(date --iso)-${TEST_SYSTEM}
```

Comment:
uber edge case but to avoid differing file/dir names if running this on midnight one would probably put the result `$(date --iso)` into a variable, and reference it here and a few line above?

Reply:
Sounds like a plan, I'll put it in a draft PR and check it all works next time I run through it!

Reply:
https://github.com/wmde/wikibase-release-pipeline/pull/372